Property 'start' does not exist on type 'Gulp' error during build.tools

<!--
IF YOU DON'T FILL OUT THE FOLLOWING INFORMATION WE MIGHT CLOSE YOUR ISSUE WITHOUT INVESTIGATING
-->

**I'm submitting a ...**  (check one with "x")
```
[x ] bug report => search github for a similar issue or PR before submitting
[ ] feature request
[ ] support request => Please do not submit support request here, instead see use [gitter](https://gitter.im/mgechev/angular2-seed) or [stackoverflow](https://stackoverflow.com/questions/tagged/angular2)
```

**Current behavior**
Execution of `gulp build.tools` from command line fails with the following error message:
```
[16:41:14] Using gulpfile ~\Documents\Angular2Dev\angular-seed-nov2016\gulpfile.ts
[16:41:14] Starting 'build.tools'...
tools\debug.ts(17,6): error TS2339: Property 'start' does not exist on type 'Gulp'.
```

**Expected behavior**
No errors.

**Minimal reproduction of the problem with instructions**
Execute `gulp build.tools` from command line.

**What is the motivation / use case for changing the behavior?**
N/A

**Please tell us about your environment:**
```
Windows 8.1 x64
$ gulp --v
 CLI version 3.9.0
 Local version 3.9.1
$ npm --v
3.10.8
```

* **Angular Seed Version:** `aaaaf75`
a9ae694   8/18/2017

* **Node:** `node --version` =
$ node -v
v6.9.1

--- a/package.json
+++ b/package.json
@@ -1,7 +1,8 @@
 {
   "name": "angular-seed",
   "version": "1.0.0",
-  "description": "Modular starter project for Angular 2 (and beyond) with statically typed build and AoT compilation",
+  "description":
+    "Modular starter project for Angular 2 (and beyond) with statically typed build and AoT compilation",
   "repository": {
     "url": "https://github.com/mgechev/angular-seed"
   },
@@ -10,39 +11,53 @@
     "build.dev.watch": "gulp build.dev.watch --color",
     "build.e2e": "gulp build.e2e --color",
     "build.prod": "gulp build.prod --color --env-config prod --build-type prod",
-    "build.prod.exp": "gulp build.prod.aot --color --env-config prod --build-type prod",
-    "build.prod.rollup.aot": "gulp build.prod.rollup.aot --color --env-config prod --build-type prod",
-    "build.prod.aot": "gulp build.prod.aot --color --env-config prod --build-type prod",
+    "build.prod.exp":
+      "gulp build.prod.aot --color --env-config prod --build-type prod",
+    "build.prod.rollup.aot":
+      "gulp build.prod.rollup.aot --color --env-config prod --build-type prod",
+    "build.prod.aot":
+      "gulp build.prod.aot --color --env-config prod --build-type prod",
     "build.test": "gulp build.test --color",
     "test.watch": "gulp test.watch --color",
     "generate.manifest": "gulp generate.manifest --color",
     "e2e": "protractor",
     "e2e.live": "protractor --elementExplorer",
     "gulp": "gulp",
-    "i18n": "gulp i18n.build --build-type prod && ng-xi18n -p dist/tmp/tsconfig.json --i18nFormat xlf && gulp i18n.merge",
+    "i18n":
+      "gulp i18n.build --build-type prod && ng-xi18n -p dist/tmp/tsconfig.json --i18nFormat xlf && gulp i18n.merge",
     "lint": "gulp tslint",
     "karma": "karma",
     "karma.start": "karma start",
-    "postinstall": "gulp check.versions && gulp build.bundle.rxjs && gulp webdriver && gulp print.banner",
+    "postinstall":
+      "gulp check.versions && gulp build.bundle.rxjs && gulp webdriver && gulp print.banner",
     "reinstall": "npm cache clean && npm install",
     "serve.coverage": "gulp serve.coverage --color",
     "serve.dev": "gulp serve.dev --color --env-config dev",
     "serve.e2e": "gulp serve.e2e --color",
     "serve.prod": "gulp serve.prod --color --env-config prod --build-type prod",
-    "serve.prod.aot": "gulp serve.prod.aot --color --env-config prod --build-type prod",
-    "serve.prod.exp": "gulp serve.prod.aot --color --env-config prod --build-type prod",
-    "serve.prod.rollup.aot": "gulp serve.prod.rollup.aot --color --env-config prod --build-type prod",
-    "sme.prod": "gulp sme.prod --color --env-config prod --build-type prod --preserve-source-maps",
-    "sme.prod.aot": "gulp sme.prod.aot --color --env-config prod --build-type prod --preserve-source-maps",
-    "sme.prod.rollup.aot": "gulp sme.prod.rollup.aot --color --env-config prod --build-type prod --preserve-source-maps",
+    "serve.prod.aot":
+      "gulp serve.prod.aot --color --env-config prod --build-type prod",
+    "serve.prod.exp":
+      "gulp serve.prod.aot --color --env-config prod --build-type prod",
+    "serve.prod.rollup.aot":
+      "gulp serve.prod.rollup.aot --color --env-config prod --build-type prod",
+    "sme.prod":
+      "gulp sme.prod --color --env-config prod --build-type prod --preserve-source-maps",
+    "sme.prod.aot":
+      "gulp sme.prod.aot --color --env-config prod --build-type prod --preserve-source-maps",
+    "sme.prod.rollup.aot":
+      "gulp sme.prod.rollup.aot --color --env-config prod --build-type prod --preserve-source-maps",
     "start": "gulp serve.dev --color",
     "start.deving": "gulp start.deving --color",
     "tasks.list": "gulp --tasks-simple --color",
     "test": "gulp test --color",
-    "e2e.ci": "gulp build.prod.rollup.aot --color && gulp build.e2e --color && gulp e2e --color",
+    "e2e.ci":
+      "gulp build.prod.rollup.aot --color && gulp build.e2e --color && gulp e2e --color",
     "tests.all": "npm test && npm run e2e.ci",
-    "webdriver-start": "node ./node_modules/protractor/bin/webdriver-manager start",
-    "webdriver-update": "node ./node_modules/protractor/bin/webdriver-manager update",
+    "webdriver-start":
+      "node ./node_modules/protractor/bin/webdriver-manager start",
+    "webdriver-update":
+      "node ./node_modules/protractor/bin/webdriver-manager update",
     "compodoc": "./node_modules/.bin/compodoc -p src/client/tsconfig.json",
     "serve.compodoc": "./node_modules/.bin/compodoc -s"
   },

--- a/tools/debug.ts
+++ b/tools/debug.ts
@@ -1,5 +1,5 @@
-import * as gulp from 'gulp';
 import { argv } from 'yargs';
+import * as runSequence from 'run-sequence';
 
 require('../gulpfile');
 
@@ -14,4 +14,4 @@ console.log('* angular-seed tools ');
 console.log('* debugging task:', TASK);
 console.log('**********************');
 
-gulp.start(TASK);
+runSequence(TASK);

--- a/tools/tasks/seed/build.tools.ts
+++ b/tools/tasks/seed/build.tools.ts
@@ -3,7 +3,7 @@ import * as gulpLoadPlugins from 'gulp-load-plugins';
 import { join } from 'path';
 
 import Config from '../../config';
-import { makeTsProject, TemplateLocalsBuilder } from '../../utils';
+import { makeTsProject } from '../../utils';
 
 const plugins = <any>gulpLoadPlugins();
 
@@ -11,7 +11,6 @@ const plugins = <any>gulpLoadPlugins();
  * Executes the build process, transpiling the TypeScript files within the `tools` directory.
  */
 export = () => {
-
   let tsProject = makeTsProject();
 
   let src = [
@@ -19,13 +18,11 @@ export = () => {
     join(Config.TOOLS_DIR, 'manual_typings/**/*.d.ts'),
     join(Config.TOOLS_DIR, '**/*.ts')
   ];
-  let result = gulp.src(src, { base: './' })
+  let result = gulp
+    .src(src, { base: './' })
     .pipe(plugins.plumber())
     .pipe(plugins.sourcemaps.init())
     .pipe(tsProject());
 
-  return result.js
-    .pipe(plugins.template(new TemplateLocalsBuilder().build()))
-    .pipe(plugins.sourcemaps.write())
-    .pipe(gulp.dest('./'));
+  return result.js.pipe(plugins.sourcemaps.write()).pipe(gulp.dest('./'));
 };

--- a/tools/tasks/seed/compile.ahead.prod.ts
+++ b/tools/tasks/seed/compile.ahead.prod.ts
@@ -3,13 +3,21 @@ import * as ts from 'typescript';
 import { argv } from 'yargs';
 import { join } from 'path';
 import { writeFileSync, readFileSync, readdirSync } from 'fs';
-import { CodeGenerator, AngularCompilerOptions, NgcCliOptions, main } from '@angular/compiler-cli';
+import {
+  CodeGenerator,
+  AngularCompilerOptions,
+  NgcCliOptions,
+  main
+} from '@angular/compiler-cli';
 
 import Config from '../../config';
 
 function codegen(
-  ngOptions: AngularCompilerOptions, cliOptions: NgcCliOptions, program: ts.Program,
-  host: ts.CompilerHost) {
+  ngOptions: AngularCompilerOptions,
+  cliOptions: NgcCliOptions,
+  program: ts.Program,
+  host: ts.CompilerHost
+) {
   return CodeGenerator.create(ngOptions, cliOptions, program, host).codegen();
 }
 
@@ -22,17 +30,25 @@ export = (done: any) => {
   // Note: dirty hack until we're able to set config easier
   modifyFile(join(Config.TMP_DIR, 'tsconfig.json'), (content: string) => {
     const parsed = JSON.parse(content);
-    const path = join(Config.PROJECT_ROOT, Config.TOOLS_DIR, 'manual_typings', 'project');
+    const path = join(
+      Config.PROJECT_ROOT,
+      Config.TOOLS_DIR,
+      'manual_typings',
+      'project'
+    );
     parsed.files = parsed.files || [];
     parsed.files = parsed.files.concat(
       readdirSync(path)
-      .filter(f => f.endsWith('d.ts'))
-      .map(f => join(path, f)));
-    parsed.files = parsed.files.filter((f: string, i: number) => parsed.files.indexOf(f) === i);
+        .filter(f => f.endsWith('d.ts'))
+        .map(f => join(path, f))
+    );
+    parsed.files = parsed.files.filter(
+      (f: string, i: number) => parsed.files.indexOf(f) === i
+    );
     parsed.files.push(join(Config.BOOTSTRAP_DIR, 'main.ts'));
     return JSON.stringify(parsed, null, 2);
   });
-  const args = argv;
+  const args = argv as any;
 
   // If a translation, tell the compiler
   if (args.lang) {

